src/customJs/payuButtonTool/PayuButtonTool.js

```diff
@@ -0,0 +1,35 @@
+import {intl} from "../localization";
+
+const React = window.unlayer.React;
+import PropTypes from 'prop-types';
+
+const baseUrl = 'https://cdn.fromdoppler.com/unlayer-editor/assets';
+
+export const PayuButtonTool = (props) => {
+  const { size, payu_detail, alignment } = props.values;
+  const buttonImgWidth = size === 'small' ? 116 : size === 'medium' ? 145 : 182;
+
+  return (
+      <div style={{
+        display: 'flex',
+        justifyContent: alignment
+      }}>
+        <a role="link" href={payu_detail ? payu_detail : ''} target="_blank">
```

Comment:
puede `payu_detail` venir vacío? en ese caso tiene sentido renderizar el boton?

Reply:
Si es necesario en situaciones de preview o en el editor, se podría poner un `#` para que el `<a>` siga siendo válido y comportandose cómo un link

```suggestion
        <a role="link" href={payu_detail ? payu_detail : '#'} target="_blank">
```

Reply:
Voy por la opción de `#` y revisaré si puedo especificar este campo como requerido en Unlayer.